app/styles/app.scss

```diff
@@ -2,41 +2,83 @@ $blue: #4c66b2;
 $white: white;
 $red: #ff0033;
 
-body{
-  color: $blue;
-  background-size: cover;
-}
-
-.colorfix{
-  background-color: $blue;
-}
-
-.header-links{
-}
-
-.btn-head{
-  width: 180px;
-  height: 100px;
-  font-size: 30px;
-}
-
-.pad-head{
-  padding: 30px 10px 0px 10px;
-}
-
-.row-head{
-  background-color: $white;
-  height: 200px;
-}
-.main_title{
-  background-color: $blue;
-  color:$white;
-  text-align: center;
-  h1{
-    color: $red;
-  }
-}
-
-.btn btn-danger, btn-primary, a, hover{
-  color: $white;
-}
+
+//==========  Mobile First Method  ==========
+
+  // Custom, iPhone Retina
+@media only screen and (min-width : 320px) {
+    .btn-danger {
+      width: 100%;
+      float: none;
+    }
+
+  }
+
+  // Extra Small Devices, Phones
+@media only screen and (min-width : 480px) {
+
+  }
+
+  // Small Devices, Tablets
+@media only screen and (min-width : 768px) {
+    .navbar-nav {
+      float: right;
+    }
+
+  }
+
+  // Medium Devices, Desktops
+@media only screen and (min-width : 992px) {
+    .btn-danger {
+      width:180px;
+      height: 100px;
+    }
+
+  }
+
+  // Large Devices, Wide Screens
+@media only screen and (min-width : 1200px) {
+
+  }
+
+body {
+    color: $blue;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-size, color, width